From: Andreas Henriksson <andreas@fatal.se>
Date: Fri, 25 Jul 2014 14:22:01 +0200
Subject: cfdisk: reenable cursor when quitting

After quitting cfdisk (built with slang) the cursor would be
missing at the command prompt.

Reported-by: Martin Steigerwald <Martin@Lichtvoll.de>
Addresses-Debian-Bug: #755991
Signed-off-by: Andreas Henriksson <andreas@fatal.se>
---
 disk-utils/cfdisk.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/disk-utils/cfdisk.c b/disk-utils/cfdisk.c
index 109489a..27da167 100644
--- a/disk-utils/cfdisk.c
+++ b/disk-utils/cfdisk.c
@@ -1966,6 +1966,7 @@ static int ui_run(struct cfdisk *cf)
 
 	menu_pop(cf);
 
+	curs_set(1);
 	DBG(FRONTEND, ul_debug("ui: end"));
 
 	return 0;
